pages/common/pio-home.md

```diff
@@ -0,0 +1,24 @@
+# pio home
+
+> Launch the PlatformIO Home Web-server.
+> More information: <https://docs.platformio.org/en/latest/core/userguide/cmd_home.html>.
+
+- Set specific HTTP port for the server (defaults to 8008):
```

Comment:
What about including a basic example without any additional arguments to start with? Generally we start with less complex examples at the top and work downwards to the more complex ones at the bottom.
Perhaps this will read better?
```suggestion
- Use a specific HTTP port (defaults to 8008):
```

Reply:
Sure, I'm on it